docs/guides/observability.md

```
@@ -0,0 +1,3 @@
+# Observability
+
+You can use OpenTelemetry in order to observe the 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)